### PR TITLE
Issue1880

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #1880  by adding a note into the Documentation on the issue 
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/1593

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ You can exclude files and folders from being evaluated by most attack surface re
 >- Block process creations originating from PSExec and WMI commands
 >- Block JavaScript or VBScript from launching downloaded executable content
 
-You can specify individual files or folders (using folder paths or fully qualified resource names) but you can't specify which rules the exclusions apply to.
+>[!IMPORTANT] The rule **Block executable files from running unless they meet a prevalence, age, or trusted list criterion** is owned by microsoft and is not specified by admins. It uses Microsoft CLoud's Protection to update its trusted list regularly.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction.md]: [!IMPORTANT] The rule **Block executable files from running unless they meet a prevalence, age, or trusted list criterion** is owned by  **Microsoft** and is not specified by admins. It uses Microsoft CLoud's Protection to update its trusted list regularly.

File: windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction.md [hunk 2]
```diff
@@ -34,7 +34,8 @@ You can exclude files and folders from being evaluated by most attack surface re
...
+>You can specify individual files or folders (using folder paths or fully qualified resource names) but you can't specify which rules or exclusions apply to.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction.md]: You can specify individual files or folders (using folder paths or fully qualified resource names) but you can't specify which rules or exclusions **to apply**.

If my understanding is correct, please transpose words.

**Comment by j0rt3g4** [comment]
Anytime @Justinha  :)

**Pull request merged**